### PR TITLE
Update credits / license attribution

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the credits / license attribution file, based on the latest code in https://github.com/OctopusDeploy/OctopusDeploy. 

Please review for
* syntactical correctness (ie, markdown looks correct)
* logical correctness (ie, when reading it, it makes sense)
* license conformance (we agree with the license in question and we are able to redistribute it)

Created by TeamCity project 'Automated Documentation Updates::Update credits.md (OSS license attribution file)', build 47

### CODE REVIEW

**Comment by matt-richardson** [comment]
Yeah - i think @shaunmarx did some front end dependency upgrades

**Comment by michaelnoonan** [comment]
@shaunmarx can you take a look at these and see if they make sense. I'll focus with @jburger on the licensing aspects.

**Comment by shaunmarx** [comment]
@michaelnoonan it does seem to be correct, although the babel in this case is a transitive dev time dependency for us. There may be some injected code from the babel runtime etc. in some of the downstream components like react-virtualized for example. Not sure if we want to provide attributions for this, but thought I would mention it regardless.

File: docs/credits.md [hunk 1]
```diff
@@ -226,7 +288,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
...
 | content-type                                      | Douglas Christopher Wilson                                                | [https://github.com/jshttp/content-type](https://github.com/jshttp/content-type)                                                                                                                                            | [MIT](https://opensource.org/licenses/MIT)                                                                          |
-| content-type-parser                               | Domenic Denicola                                                          | [https://github.com/jsdom/content-type-parser](https://github.com/jsdom/content-type-parser)                                                                                                                                | [WTFPL](http://www.wtfpl.net/)                                                                                      |
```
Comment (jburger) [inline docs/credits.md]: :tada: happy this one is gone

File: docs/credits.md [hunk 2]
```diff
@@ -547,26 +594,26 @@ Octopus Deploy is made possible thanks to the following great third-party produc
...
 | lcid                                              | Sindre Sorhus                                                             | [https://github.com/sindresorhus/lcid](https://github.com/sindresorhus/lcid)                                                                                                                                                | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | left-pad                                          | azer                                                                      | [https://github.com/stevemao/left-pad](https://github.com/stevemao/left-pad)                                                                                                                                                | [WTFPL](http://www.wtfpl.net/)                                                                                      |
```
Comment (jburger) [inline docs/credits.md]: ...almost

**Pull request merged**